interfaces/builtin/ppp.go

```diff
@@ -41,11 +41,6 @@ var pppConnectedPlugAppArmor = []byte(`
 @{PROC}/@{pid}/loginuid r,
 capability setgid,
 capability setuid,
-/sbin/resolvconf rix,
-/run/resolvconf** rw,
-/etc/resolvconf/** rw,
-/etc/resolvconf/update.d/* ix,
-/lib/resolvconf/* ix,
```

Comment:
Why are these being removed from the ppp interface? It's fine to add them to network-manager, but will pppd ever need this or do we expect that a 'ppp without network-manager snap' would be required to use 'network-control'? If pppd does actually need it, please add back with the additions I gave above, but if it is to be used with network-control, it's fine to remove.